### PR TITLE
Update LibreWolf.LibreWolf to v100.0.1

### PR DESCRIPTION
**Pull request by maltejur**
This pull-request was auto-generated by [maltejur/librewolf-winget](https://github.com/maltejur/librewolf-winget) and manualy triggered by @stanzabird

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59585)

### CODE REVIEW

File: manifests/l/LibreWolf/LibreWolf/100.0/LibreWolf.LibreWolf.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+PackageIdentifier: LibreWolf.LibreWolf
+PackageVersion: 100.0
```
Comment (Trenly) [inline manifests/l/LibreWolf/LibreWolf/100.0/LibreWolf.LibreWolf.installer.yaml:2]:
```suggestion
PackageVersion: 100.0.1
```

File: manifests/l/LibreWolf/LibreWolf/100.0/LibreWolf.LibreWolf.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+PackageIdentifier: LibreWolf.LibreWolf
+PackageVersion: 100.0
```
Comment (Trenly) [inline manifests/l/LibreWolf/LibreWolf/100.0/LibreWolf.LibreWolf.locale.en-US.yaml:2]:
```suggestion
PackageVersion: 100.0.1
```

File: manifests/l/LibreWolf/LibreWolf/100.0/LibreWolf.LibreWolf.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+PackageIdentifier: LibreWolf.LibreWolf
+PackageVersion: 100.0
```
Comment (Trenly) [inline manifests/l/LibreWolf/LibreWolf/100.0/LibreWolf.LibreWolf.yaml:2]:
```suggestion
PackageVersion: 100.0.1
```

**Review by Trenly — CHANGES_REQUESTED** [review]
The PackageVersion must match control panel; When updating this, you will need to update the name of the version folder also. You can do this in GitHub web by pressing <kbd>.</kbd>

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**